src/edg.rs

```diff
@@ -417,7 +567,23 @@ impl<B: Broker<V> + Debug, G: ExtendedDependencyGraph<V> + Send + Sync + Debug,
                         },
                     );
                 }
-                VertexAssignment::UNDECIDED => {} // fallthrough
+                _ => {
+                    // update distance
+                    let dist = self.distances.get(vertex).unwrap_or(&0);
+                    self.distances
+                        .insert(vertex.clone(), max(dist.clone(), distance));
```

Comment:
Fixes `clippy` warning
```suggestion
                        .insert(vertex.clone(), max(dist, distance));
```